app/pb_kits/playbook/pb_fixed_confirmation_toast/_fixed_confirmation_toast.jsx

```diff
@@ -14,36 +14,51 @@ const iconMap = {
 
 type FixedConfirmationToastProps = {
   className?: string,
+  closeable?: boolean,
   data?: string,
   id?: string,
   status?: "success" | "error" | "neutral" | "tip",
   text: string,
 }
 
 const FixedConfirmationToast = (props: FixedConfirmationToastProps) => {
-  const { className, status = 'neutral', text } = props
+  const [showToast, toggleToast] = useState(true)
+  const { className, closeable = false, status = 'neutral', text } = props
   const css = classnames(
     `pb_fixed_confirmation_toast_kit_${status}`,
     globalProps(props),
-    className
+    className,
+    { 'remove_toast': closeable }
```

Comment:
yeah className needs to go last here. 

So i would do:
```
   { 'remove_toast': closeable },
    globalProps(props),
    className
```

Reply:
and github was yelling at me about suggesting the commit-- hence i didn't do it.